src/test_l1_dsDisplay.c

```diff
@@ -98,6 +100,19 @@ static int gTestID = 1;
     }\
 }\
 
+#define CHECK_FOR_EXTENDED_ERROR_CODE( result, enhanced, old )\
```

Comment:
#defines have GLOBAL Scope, you'll be re-using the same macro in all files..

therefore has to be module specific

`DS_DISPLAY_CHECK_FOR_EXTEND_ERROR_CODE`

it's only working by luck.